lib/oxidized/model/fortios.rb

```diff
@@ -15,8 +15,10 @@ class FortiOS < Oxidized::Model
   end
 
   cmd :secret do |cfg|
-    cfg.gsub! /(set (?:passwd|password|psksecret|secret|key ENC)).*/, '\\1 <configuration removed>'
+    cfg.gsub! /(ENC ).*/, '\\1 <configuration removed>'
```

Comment:
How come the original line was changed to this?

Reply:
I changed it because in the original cfg.gsub you need to specify all string to match...
This string can match all the occurance

Reply:
Is their not a risk of this then matching more lines?

Reply:
Hi Neil... Sorry for the delay in my answer, but i'm very busy with my job.
In my configuration it match correct all the lines.
Here an example of line match in 1 of our fortigate


set password ENC <configuration removed>
set password ENC <configuration removed>
set history0 ENC <configuration removed>
set history1 ENC <configuration removed>
set password ENC <configuration removed>
set history0 ENC <configuration removed>
set history1 ENC <configuration removed>
set history0 ENC <configuration removed>
set history1 ENC <configuration removed>
set password ENC <configuration removed>
set password ENC <configuration removed>
set password ENC <configuration removed>
set password ENC <configuration removed>
set password ENC <configuration removed>
set password ENC <configuration removed>
set password ENC <configuration removed>
set password ENC <configuration removed>
set group-password ENC <configuration removed>
set passwd1 ENC <configuration removed>
set passwd2 ENC <configuration removed>
set passwd3 ENC <configuration removed>
set secret ENC <configuration removed>
set secondary-secret ENC <configuration removed>
set tertiary-secret ENC <configuration removed>
set passwd ENC <configuration removed>
set auth-password-l1 ENC <configuration removed>
set auth-password-l2 ENC <configuration removed>
set password ENC <configuration removed>
set password ENC <configuration removed>
set password ENC <configuration removed>
set password ENC <configuration removed>
set password ENC <configuration removed>
set password ENC <configuration removed>
set password ENC <configuration removed>
set password ENC <configuration removed>
set password ENC <configuration removed>
set password ENC <configuration removed>
set password ENC <configuration removed>
set password ENC <configuration removed>
set psksecret ENC <configuration removed>
set psksecret ENC <configuration removed>
set auth-password-l1 ENC <configuration removed>
set auth-password-l2 ENC <configuration removed>

Simone

Reply:
Yeah but will also match interface descriptions with ENC in it.

Reply:
Hi Neil, we have a good number of forti, e no one of them have ENC in some description... But it's possible someone have it,... I'll change it in the fortios.rb ... Just in case ..

Reply:
@laf 

All Done